libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKIDPAuthClient.m

```diff
@@ -0,0 +1,414 @@
+/*
+ SFSDKIDPAuthClient.m
```

Comment:
Could we add comments on each method to explain how one gets there (and what happens next)?